<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>771</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>创建数据</string>
    </property>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>客户登记</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>合同登记</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>合同优惠单登记</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>开工预案（UI）</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>设计见面</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>报价单（UI）</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>开工申请（UI）</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>核算单（UI）</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>54</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>新宋体</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>数量：</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>54</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>新宋体</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>账户：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_15">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>30</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>30</y>
       <width>54</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>新宋体</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_16">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>30</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>54</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>新宋体</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>结果：</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>531</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>771</width>
      <height>481</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>IMS账户登录</string>
    </property>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Master</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Release_1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Release_2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Release_3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Release_4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Release_5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Release_6</string>
      </property>
     </item>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>171</width>
       <height>421</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>仿宋</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>毛伟人</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_8">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>仿宋</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TEST</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_7">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>仿宋</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>幸福毛经理</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>仿宋</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>幸福副总</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>仿宋</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>工种经理</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_13">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>仿宋</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>小陈装修管家</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_15">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>仿宋</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>苏轼</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_16">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>仿宋</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>备用3</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_17">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>仿宋</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>备用4</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_3">
        <property name="font">
         <font>
          <family>仿宋</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>账户1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>账户2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>账户3</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>71</width>
       <height>421</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>附加页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_6">
        <property name="text">
         <string>附加页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_8">
        <property name="text">
         <string>附加页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_10">
        <property name="text">
         <string>附加页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_11">
        <property name="text">
         <string>附加页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_5">
        <property name="text">
         <string>附加页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_13">
        <property name="text">
         <string>附加页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_3">
        <property name="text">
         <string>附加页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_15">
        <property name="text">
         <string>附加页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_14">
        <property name="text">
         <string>附加页</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>60</y>
       <width>501</width>
       <height>421</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLineEdit" name="lineEdit_12"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_11"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_5"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_6"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_7"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_8"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_9">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>771</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>日常应用工具</string>
    </property>
    <widget class="QPushButton" name="pushButton_6">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>20</y>
       <width>171</width>
       <height>35</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>仿宋</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>数据库</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>60</y>
       <width>171</width>
       <height>35</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>仿宋</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fiddler</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>60</y>
       <width>171</width>
       <height>35</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>仿宋</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>更新IMS</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>171</width>
       <height>35</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>仿宋</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>模拟器</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>171</width>
       <height>35</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>仿宋</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Pycharm</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_14">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>171</width>
       <height>35</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>仿宋</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Obsidian</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>主菜单</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
